### ISSUE TITLE
📙 Clarification about Parse Server installation methods

### ISSUE THREAD

**Issue by MrMartinR** [ParsePlatform/Docs#878]
### New Feature / Enhancement Checklist

- [X] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [X] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [X] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation
Confusing about Installation

### Feature / Enhancement Description
Be more clear about the different installation options / unify the info in one place 

### Example Use Case
I think I mess up my installation because I did install parse-server from npm and also I did run the bootstrap file, and I think I have 2 parse servers, 
I think I was running the bootstrap version but I had some drama when I tried to add parse server to PM2, adding environment variables, etc..

Also I need clarification about this, because I was editing the docs and In the [ubuntu deployment](https://docs.parseplatform.org/parse-server/guide/#deploying-on-aws-ec2-ubuntu-using-postgresql) I stated to perform both installations (via npm and via bootstrap)

Also the files and folder structure are not the same
<img width="412" alt="Screen Shot 2022-06-11 at 17 06 03" src="https://user-images.githubusercontent.com/64530191/173198584-41dcd66b-3618-45cd-903b-1a0cf7c54c2e.png">

btw: the key generations in the bootstrap script doesn't work at least in ubuntu

### Alternatives / Workarounds
Learn more about devOps

### 3rd Party References
none

**Comment by mtrezza** [comment]
Transferring to docs, since this seems to be a docs issue, not a product issue.

**Comment by mtrezza** [comment]
> I think I mess up my installation because I did install parse-server from npm and also I did run the bootstrap file, and I think I have 2 parse servers,

Where did you see an instruction to execute both?

Looking at the Parse Server [README](https://github.com/parse-community/parse-server#locally) I see only this:

```
$ npm install -g parse-server mongodb-runner
$ mongodb-runner start
$ parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --databaseURI mongodb://localhost/test
```

**Comment by MrMartinR** [comment]
I haven't see both in the same place, I saw the npm version in the README and the Bootstrap version in the Docs Getting Started Section.
`Use the bootstrap script to set up Parse Server in the current directory.`

I assumed? that maybe the npm was some library that parse-server needed and the "official" parse server installation was the Bootstrap version that maybe also installed the npm... 🤷‍♂️
Or maybe I thought that the npm was 'the one' and the Bootstrap was the parse-server I need to clone/fork to contribute to the project 🤷‍♂️ I just installed both🤦‍♂️

To be honest, no idea what is the difference, are they the same?

**Comment by mtrezza** [comment]
The bootstrap script installs parse server - and more. You can take a look at the script to see whether it's useful for you. The `npm i -g parse-server` installs parse server globally, which makes it accessible from anywhere in the CLI but may make it more difficult for Parse Server development when switching between versions of Parse Server frequently.

What is right for you depends on what you want to do. Either guide works, but with different implications.

I agree that this may seem confusing for new developers who are just getting started with Node.js development. A proper guide should consolidate and offer both approaches to set up Parse Server and explain the differences. But it doesn't make any sense to duplicate that information in the README and the docs. The README should be stripped of all docs-like content and that content should move into the docs.

If you want to look more into this, see the npm docs link I posted above and study the bootstrap script to see what it does. It may be a bit of work, but the learning and insight should be worth it. And maybe you want to open a PR afterwards to improve the docs in that regard.

**Comment by MrMartinR** [comment]
I agree 100% that the README (and the wiki) should not contain any doc related stuff... I am gonna try to dig into those 2 options and edit the AWS Deployment Guide (it states to do both installs)...
probably more questions will pop in my head, should I come back here? or use Slack??

**Comment by mtrezza** [comment]
Great, it's best to discuss docs changes here, since it has more visibility and allows for collaboration. This issue is just to clarify the installation installations in the docs. If your changes are broader, then I suggest opening a separate issue.

**Comment by MrMartinR** [comment]
I did refactor the AWS guide, I decided to install Parse Dashboard Globally and Parse-Server locally but without using the Bootstrap script, after checking the bootstrap I think is not worth to use it in the AWS deployment, only generates a couple of extra files...
also I fix the block codes from ```js to ```json
